package.json

```diff
@@ -91,11 +91,11 @@
     "@emotion/react": "^11.13.3",
     "@emotion/styled": "^11.13.0",
     "@mui/icons-material": "^5.16.13",
-    "@mui/internal-babel-plugin-resolve-imports": "1.0.18",
-    "@mui/internal-markdown": "^1.0.19",
+    "@mui/internal-babel-plugin-resolve-imports": "1.0.20",
+    "@mui/internal-markdown": "^1.0.25",
     "@mui/internal-test-utils": "^1.0.24",
     "@mui/material": "^5.16.13",
-    "@mui/monorepo": "github:mui/material-ui#356d5dffbcbc1c18b5ab8060a173ea7a3e90b58c",
+    "@mui/monorepo": "github:mui/material-ui#059a46c9d3e1731aab792eb7cfa0d5788c91cebe",
```

Comment:
I assume you need to merge the PR before to get the correct commit hash

Reply:
Need - probably not, should - yes. 👌

Reply:
Done.